### DISCUSSION TITLE
time_format Configuration

### DISCUSSION THREAD

**Discussion by stuartle** [Serene-Arc/bulk-downloader-for-reddit#960] [Q&A] [answered]
The general page doesn't go into detail on how to configure {DATE} using the time_format definition. Simply want the date for be formatted as MMDDYYYY. Is this functionality not yet added?

**Comment by Serene-Arc** [comment, marked as answer]
The formats given use the standard Python format codes, as detailed in the README. For what you want, the code would be `%m%d%Y`.
